python/Framework.py

```diff
@@ -509,23 +509,8 @@ def doSystematics(self, systematics, **kwargs):
     def configureElectronId_(self):
 
         with StdStreamSilenter():
-            from PhysicsTools.SelectorUtils.tools.vid_id_tools import DataFormat, switchOnVIDElectronIdProducer, setupAllVIDIdsInModule, setupVIDElectronSelection
-            switchOnVIDElectronIdProducer(self.process, DataFormat.MiniAOD)
-
-            # Use in input our own electorn collection
-            self.process.egmGsfElectronIDs.physicsObjectSrc = self.__miniaod_electron_collection
-
-            id_modules = [
-                    'RecoEgamma.ElectronIdentification.Identification.cutBasedElectronID_Summer16_80X_V1_cff',
-                    'RecoEgamma.ElectronIdentification.Identification.cutBasedElectronHLTPreselecition_Summer16_V1_cff',
-                    'RecoEgamma.ElectronIdentification.Identification.mvaElectronID_Spring16_GeneralPurpose_V1_cff',
-                    'RecoEgamma.ElectronIdentification.Identification.mvaElectronID_Spring16_HZZ_V1_cff'
-                    ]
-
-            for mod in id_modules:
-                setupAllVIDIdsInModule(self.process, mod, setupVIDElectronSelection)
-
-            self.process.electronMVAValueMapProducer.srcMiniAOD = self.__miniaod_electron_collection
+          self.process.load("RecoEgamma.Phase2InterimID.phase2EgammaPAT_cff")
+          self.path += cms.Sequence(self.process.phase2Egamma)
```

Comment:
There's nothing wrong with adding the module to the path explicitly, but we use "on-demand scheduling" everywhere for the framework (this changed in 91 or 92, you now need to add "tasks"), could you try and see if `self.path.associate(cms.Task(self.process.phase2Egamma))` also works?

Reply:
----- Begin Fatal Exception 23-Mar-2018 16:33:50 CET-----------------------
An exception of category 'ConfigFileReadError' occurred while
   [0] Processing the python configuration file named HHConfiguration.py
Exception Message:
python encountered the error: <type 'exceptions.RuntimeError'>
Adding an entry of type 'Sequence'to a Task.
It is illegal to add this type to a Task.
----- End Fatal Exception -------------------------------------------------

Reply:
Let's leave it like this for now then (this will come back when you add the systematics)